### ISSUE TITLE
Rultor responds slowly

### ISSUE THREAD

**Issue by sils** [yegor256/rultor#898]
Hey, you state, rultor responds within 60 seconds. There's probably a good technical reason for this but I'd still love to ask if this can somehow be reduced, after all devs are waiting for this time (at least sometimes to see if it works) and scraping off a few seconds from the workflow here and there would be cool.

**Comment by yegor256** [comment]
@sils1297 we can try to make it faster by processing all "talks" in parallel. Now they go one by one, sequentially. I'll try to fix it.. thanks

**Comment by alex-palevsky** [comment]
@sils1297 I will ask someone to take care of this task soon

**Comment by alex-palevsky** [comment]
@sils1297 I set the milestone to 2.0 since there is nothing set yet

**Comment by alex-palevsky** [comment]
@yegor256 it is yours, help us

**Comment by yegor256** [comment]
@alex-palevsky assign someone else here pls

**Comment by alex-palevsky** [comment]
> @alex-palevsky assign someone else here pls

@yegor256 -30 points to your rating

**Comment by alex-palevsky** [comment]
> @alex-palevsky assign someone else here pls

@yegor256 I will ask somebody else to pick this up

**Comment by original-brownbear** [comment]
@alex-palevsky this is postponed.

**Comment by alex-palevsky** [comment]
> @alex-palevsky this is postponed.

@original-brownbear thanks, I added "postponed" label

**Comment by alex-palevsky** [comment]
> @alex-palevsky this is postponed.

@original-brownbear right, I will find someone else, no problem

**Comment by 0crat** [comment]
I can't do it for technical reasons, I'm very sorry. If you don't know what to do, submit this error as a ticket [here](https://github.com/zerocracy/datum):

```
1.0-SNAPSHOT 0d912b4 2017-12-27T21:47:39
IllegalStateException: java.lang.reflect.InvocationTargetException in com/zerocracy/stk/pm/in/orders/finish_order.groovy
java.lang.IllegalStateException: java.lang.reflect.InvocationTargetException in com/zerocracy/stk/pm/in/orders/finish_order.groovy
	at com.zerocracy.farm.reactive.StkGroovy.process(StkGroovy.java:125)
	at com.zerocracy.farm.StkSafe.process(StkSafe.java:87)
	at com.zerocracy.farm.reactive.StkSmart.process(StkSmart.java:66)
	at com.zerocracy.farm.reactive.Brigade.process(Brigade.java:82)
	at com.zerocracy.farm.reactive.Brigade.lambda$apply$0(Brigade.java:64)
	at org.cactoos.func.UncheckedFunc.lambda$apply$0(UncheckedFunc.java:58)
	at org.cactoos.func.IoCheckedBiFunc.apply(IoCheckedBiFunc.java:66)
	at org.cactoos.func.UncheckedBiFunc.apply(UncheckedBiFunc.java:60)
	at org.cactoos.func.UncheckedFunc.apply(UncheckedFunc.java:59)
	at org.cactoos.iterator.Filtered.hasNext(Filtered.java:90)
	at org.cactoos.iterator.LengthOf.value(LengthOf.java:56)
	at org.cactoos.iterable.LengthOf.value(LengthOf.java:56)
	at com.zerocracy.farm.reactive.Brigade.apply(Brigade.java:67)
	at com.zerocracy.farm.r...
```

Cc @yegor256

**Comment by 0crat** [comment]
Oops! Job `gh:yegor256/rultor#898` was not in scope
